composer.json

```diff
@@ -24,7 +24,7 @@
         "illuminate/support": "^9.0"
     },
     "require-dev": {
-        "meilisearch/meilisearch-php": "^0.23",
+        "meilisearch/meilisearch-php": ">=0.26.1 <1.0 | ^1.0",
```

Comment:
@driesvints I added this version constraint, so that tests are still possible. As soon as meilisearch and meilisearch-php hit v1. we could set this to `^1.0` only

Reply:
upgraded the version to v1 now so this is rdy for when meilisearch-php v1 is released